test/natalie/rbconfig_test.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../spec_helper'
+require 'rbconfig'
+
+describe 'RbConfig::CONFIG' do
+  platform_is :linux do
+    it 'uses the correct values for DLEXT and SOEXT' do
+      RbConfig::CONFIG['DLEXT'].should == 'so'
+      RbConfig::CONFIG['SOEXT'].should == 'so'
+    end
+  end
+
+  platform_is :darwin do
+    it 'uses the correct values for DLEXT and SOEXT' do
+      RbConfig::CONFIG['DLEXT'].should == 'bundle'
+      RbConfig::CONFIG['SOEXT'].should == 'dylib'
+    end
+  end
```

Comment:
This is the main reason for adding this test: this runs in MRI as well, so this way I can validate I didn't mix up those two values.